docs/plugin-config.md

```diff
@@ -0,0 +1,76 @@
+# Plugin Configuration
+The `notation-azure-kv` plugin supports plugin configuration to use advanced features.
```

Comment:
For `notation-azure-kv` plugin, all the plugin configs are for one purpose, i.e., `get a valid cert chain or self-signed cert during remote signing`. Let's make it clear in the doc.

Reply:
Thanks! Added.